test/integrations/destinations/facebook_pixel/dataDelivery/data.ts

```diff
@@ -41,21 +41,14 @@ export const data = [
           output: {
             status: 400,
             message: 'Invalid OAuth 2.0 access token',
-            destinationResponse: {
-              error: {
-                message: 'The access token could not be decrypted',
-                type: 'OAuthException',
-                code: 190,
-                fbtrace_id: 'fbpixel_trace_id',
-              },
-              status: 500,
-            },
+            destinationResponse: '',
```

Comment:
Let's enhance the integrations-lib's class to address the ConfigurationAuth error. Specifically, when someone sends a destinationResponse, we should ensure that it includes relevant information.

Please let me know if this clarification is clear or if you have any questions.

Reply:
```
export class ConfigurationAuthError extends BaseError {
    constructor(message: string, destResponse: any | undefined = undefined) {
        const finalStatTags = {
            [TAG_NAMES.ERROR_CATEGORY]: ERROR_CATEGORIES.DATA_VALIDATION,
            [TAG_NAMES.ERROR_TYPE]: ERROR_TYPES.CONFIGURATION,
            [TAG_NAMES.META]: ERROR_TYPES.AUTH,
        };
        super(message, 400, finalStatTags, destResponse);
    }
}
```
From other error categories I see we are passing the entire destination response. Is it what you are asking to do here as well?

Reply:
added destination response as discussed